network_internal.conf

```diff
@@ -3,4 +3,5 @@ allow 127.0.0.0/8;
 allow 10.0.0.0/8;
 allow 192.168.0.0/16;
 allow 172.16.0.0/12;
+allow fc00::/7; # IPv6 local address range
```

Comment:
Should also include `::1/128` loopback IPv6 (there's only one, in comparison to IPv4's huge loopback block)

Reply:
I am not sure if we need any loopback at all. Loopback in this scenario would mean that the proxy container itself makes a request to localhost. It could be added.